### PR TITLE
JDF833: Plugin commands have changed from jboss-as prefix to wildfly …

### PR DESCRIPTION
**Pull request by sgilda**
…prefix. jboss-as:deploy is now wildfly:deploy and jboss-as:undeploy is now wildfly:undeploy.

### CODE REVIEW

File: cluster-ha-singleton/README.md [hunk 1]
```diff
@@ -76,16 +76,16 @@ Build and Deploy the Quickstart
...
 
 4. This deploys `service/target/jboss-cluster-ha-singleton-service.jar` to the running instance of the first server.
 5. Since default socket binding port is `9999` and the second server runs at a port offset of `100`, you must pass port `10099` (9999 + 100) as an argument when you deploy to the second server. Type this command to deploy the archive to the second server. 
 
-        mvn jboss-as:deploy -Djboss-as.port=10099
+        mvn wildfly:deploy -Djboss-as.port=10099
     
     If the second server is on a different host, you must also pass an argument for the host name as follows:
     
-        mvn jboss-as:deploy [-Djboss-as.hostname=OTHERHOST] -Djboss-as.port=10099
```
Comment (rafabene) [inline cluster-ha-singleton/README.md:88]: The property have changed from jboss-as.port to wildfly.port. See: http://docs.jboss.org/wildfly/plugins/maven/latest/deploy-mojo.html#port

**Comment by rafabene** [comment]
Just a minor detail as I commented above and it's good to go.

File: cluster-ha-singleton/README.md (hunk 1 above)
Comment (sgilda) [inline cluster-ha-singleton/README.md:88]: @rafabene : It looks like he hostname has also changed. :-)

Comment (rafabene) [inline cluster-ha-singleton/README.md:88]: Good catch :+1:

Comment (rafabene) [inline cluster-ha-singleton/README.md:88]: -Dwildfly.hostname

**Comment by sgilda** [comment]
@rafabene : Changes are made. Do you want to review or should I just go for it?

**Comment by rafabene** [comment]
I don't need to review it again :). Thanks

**Comment by sgilda** [comment]
Closed due to too many conflicts. Opened a new pull instead: https://github.com/jboss-developer/jboss-eap-quickstarts/pull/1429

**Pull request closed without merging**